lib/utils/forkedblockchain.js

```diff
@@ -29,10 +30,11 @@ function ForkedBlockchain(options) {
   this.forkVersion = null;
 
   if (typeof options.fork === "string") {
-    if (options.fork.indexOf("@") >= 0) {
-      const split = options.fork.split("@", 2);
-      options.fork = split[0];
-      options.fork_block_number = parseInt(split[1]);
+    const blockNumber = blockNumberReg.exec(options.fork);
+
+    if (blockNumber) {
+      options.fork = options.fork.replace(blockNumber, "");
+      options.fork_block_number = parseInt(blockNumber.pop().replace("@", ""));
```

Comment:
You you mind updating the regex to:

```javascript
const blockNumberReg = /@([0-9]+)$/;
````

and then the `fork` and `fork_block_number` assignment to:

```javascript
options.fork = options.fork.slice(0, blockNumber.index);
options.fork_block_number = parseInt(blockNumber[1], 10);
```

Reply:
Hey @davidmurdoch! thanks for the review. I've pushed the changes, looking forward to the merge!